src/seamless-immutable.js

```diff
@@ -5,11 +5,36 @@
   var REACT_ELEMENT_TYPE = typeof Symbol === 'function' && Symbol.for && Symbol.for('react.element');
   var REACT_ELEMENT_TYPE_FALLBACK = 0xeac7;
 
+  function instantiateEmptyObject(obj) {
+      var prototype = Object.getPrototypeOf(obj);
+      if (!prototype) {
+          return {};
+      } else {
+          return Object.create(prototype);
+      }
+  }
+
+  function staticWarning(fn, name) {
+      function staticWrapper() {
+          console.log('DEPRECATED: Using x.' + name + '(...) will be deprecated. Please use Immutable.' + name + '(x, ...) instead.');
+          return fn.apply(this, [].slice.call(arguments));
+      }
+
+      if (process.env.NODE_ENV !== "production") {
```

Comment:
Will this blow up in the browser? Not sure what the build system for this project is like